source/extensions/filters/http/cache/simple_http_cache/simple_http_cache.cc

```diff
@@ -305,7 +305,7 @@ CacheInfo SimpleHttpCache::cacheInfo() const {
   return cache_info;
 }
 
-SINGLETON_MANAGER_REGISTRATION(simple_http_cache_singleton);
+SINGLETON_MANAGER_REGISTRATION(simple_http_cache);
```

Comment:
I'd avoid renaming preexisting variables unnecessarily, when the code belongs to someone else.